src/hipscat/io/write_metadata.py

```diff
@@ -106,8 +106,23 @@ def write_partition_info(catalog_parameters, destination_pixel_map: dict):
         catalog_parameters.catalog_base_dir
     )
     data_frame = pd.DataFrame(destination_pixel_map.keys())
-    data_frame.columns = ["order", "pixel", "num_objects"]
-    data_frame = data_frame.astype(int)
+
+    data_frame.columns = [
+        "Norder",
+        "Npix",
+        "num_objects",
+    ]
+    data_frame["Dir"] = int(data_frame["Npix"] / 10_000) * 10_000
+
```

Comment:
I think this just reorders the columns and forces the type to int, right? Might be worth a comment to mention that.